api/api.go

```diff
@@ -74,33 +74,33 @@ var (
 	ErrAction = errors.New("invalid action")
 )
 
-// BroadcastOutbound sends a broadcast message to the whole network
-type BroadcastOutbound func(ctx context.Context, chainID uint32, msg proto.Message) error
-
-// Config represents the config to setup api
-type Config struct {
-	broadcastHandler  BroadcastOutbound
-	electionCommittee committee.Committee
-}
-
-// Option is the option to override the api config
-type Option func(cfg *Config) error
-
-// WithBroadcastOutbound is the option to broadcast msg outbound
-func WithBroadcastOutbound(broadcastHandler BroadcastOutbound) Option {
-	return func(cfg *Config) error {
-		cfg.broadcastHandler = broadcastHandler
-		return nil
-	}
-}
-
-// WithNativeElection is the option to return native election data through API.
-func WithNativeElection(committee committee.Committee) Option {
-	return func(cfg *Config) error {
-		cfg.electionCommittee = committee
-		return nil
-	}
-}
+// // BroadcastOutbound sends a broadcast message to the whole network
```

Comment:
just remove unused code, don't do this

Reply:
removed. Once the test has passed, the file `api.go` and `api_test.go` will be deleted in the next PR